doc/ref/plotting_options/resampling.ipynb

```diff
@@ -19,7 +19,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The `hvsampledata.synthetic_clusters` dataset is in many examples below."
+    "The `hvsampledata.synthetic_clusters` dataset is used in many examples below."
```

Comment:
Could you write a brief description of the dataset? I think this would complement the code below nicely.

Reply:
Done in https://github.com/holoviz/hvplot/pull/1585/commits/185f6ca970580369286adaedbf49d60a7a52273f